### ISSUE TITLE
Repeated field documentation is overwritten by fields comments

### ISSUE THREAD

**Issue by fahernandez** [grpc-ecosystem/grpc-gateway#863]
### Bug report:

When defining repeated fields the documentation of the field is overwritten by the fields comments. For example:

```
// Some field comments
repeated string names = 2 [(grpc.gateway.protoc_gen_swagger.options.openapiv2_field) =
    {
        description:
            "Some field description documentation."
           array: ["name1", "name2"];
    }];
```
After generating the swagger documentation the field comments takes precendence over the field documentation resulting in something like:

```
{
        "name": "names",
        "description": "Some field comments",
        "in": "query",
        "required": false,
        "type": "array",
        "items": {
             "type": "string"
         }
}
```

I think that the field documentation must take precedence over the field comments.

**Comment by achew22** [comment]
This would be a really great starter project for getting involved!

This comes from the interaction of these two pieces of code.

https://github.com/grpc-ecosystem/grpc-gateway/blob/c3923b1034cd4a10cea4dfae173804b19f6f7fe0/protoc-gen-swagger/genswagger/template.go#L120-L124

https://github.com/grpc-ecosystem/grpc-gateway/blob/c3923b1034cd4a10cea4dfae173804b19f6f7fe0/protoc-gen-swagger/genswagger/template.go#L294-L296

A little bit of rejiggering in there + a new test case and I think you'd be done if you're interested.

**Comment by fahernandez** [comment]
Sure, I will definitely try, thank you for taking me in the right direction!

**Comment by johanbrandhorst** [comment]
Closed by #863
